ksp_plugin/pile_up.cpp

```diff
@@ -414,35 +451,16 @@ void PileUp::AppendToPart(DiscreteTrajectory<Barycentric>::Iterator it) const {
   auto const pile_up_to_barycentric = barycentric_to_pile_up.Inverse();
   for (not_null<Part*> const part : parts_) {
     DegreesOfFreedom<NonRotatingPileUp> const actual_part_degrees_of_freedom =
-        FindOrDie(actual_part_rigid_motion_, part)({RigidPart::origin,
-                                                    RigidPart::unmoving});
+        FindOrDie(actual_part_rigid_motion_,
+                  part)({RigidPart::origin, RigidPart::unmoving});
     (static_cast<Part*>(part)->*append_to_part_trajectory)(
-        it->time,
-        pile_up_to_barycentric(actual_part_degrees_of_freedom));
-  }
-}
-
-void PileUp::RecomputeFromParts(
-    std::list<not_null<Part*>> const& parts) {
-  mechanical_system_ =
-      std::make_unique<MechanicalSystem<Barycentric, NonRotatingPileUp>>();
-  intrinsic_force_ = Vector<Force, Barycentric>();
-
-  // TODO(phl): We assume that forces are applied at the centre of mass of the
-  // pile-up, but they are really applied at some unknown point of the parts, so
-  // this introduces a torque.
-  for (not_null<Part*> const part : parts) {
-    intrinsic_force_ += part->intrinsic_force();
-    mechanical_system_->AddRigidBody(part->rigid_motion(),
-                                     part->mass(),
-                                     part->inertia_tensor());
+        it->time, pile_up_to_barycentric(actual_part_degrees_of_freedom));
   }
 }
 
 PileUpFuture::PileUpFuture(not_null<PileUp const*> const pile_up,
                            std::future<Status> future)
-    : pile_up(pile_up),
-      future(std::move(future)) {}
+    : pile_up(pile_up), future(std::move(future)) {}
```

Comment:
Please revert, we mostly try to keep one member per line.  In general, all this reformatting is not useful.